includes/html/functions.inc.php

```diff
@@ -1328,6 +1332,21 @@ function get_arrays_with_seafile($device, $app_id, $category)
     return get_arrays_with_application($device, $app_id, $app_name, $category);
 }
 
+/**
+ * Get all dhcp data from the collected
+ * rrd files.
+ *
+ * @param array $device device for which we get the rrd's
+ * @param int   $app_id application id on the device
+ * @param string $category which category of dhcp graphs are searched
+ * @return array list of dhcp data
+ */
+function get_arrays_with_dhcpstats($device, $app_id, $category)
```

Comment:
This function is not worth anything right?  Just hardcodes a string...

Reply:
just stayed in 'structure' 
-> certificates, seafile, mdadm, smart ... + dhcpstats

Reply:
no need to copy bad code :)  just call get_arrays_with_application()

Reply:
removed ;-)